.github/workflows/conformance-ingress.yaml

```diff
@@ -36,8 +38,21 @@ jobs:
         include:
         - name: Without XDP
           bpf-lb-acceleration: disabled
+          loadbalancer-mode: dedicated
+          default-ingress-controller: false
         - name: With XDP
           bpf-lb-acceleration: native
+          loadbalancer-mode: dedicated
+          default-ingress-controller: false
+        - name: With Shared LB
+          bpf-lb-acceleration: disabled
+          loadbalancer-mode: shared
+          default-ingress-controller: false
+        - name: With Default Ingress Controller
```

Comment:
Do we somewhere test behaviour when `tproxy` is in bpf mode? Learned from #24011 that it does impact our l7 proxying a lot

Reply:
ah Ingress and GatewayAPI is relying on kpr as strict, so tproxy is already enabled afaik.

Reply:
(clarified offline that it is not tested but is out of scope of this PR)